ConsentViewController/Classes/GDPRConsentViewController.swift

```diff
@@ -57,6 +57,16 @@ typealias Meta = String
         }
     }
 
+    // Default Message/PM language set to English
+    var messageLanguage: MessageLanguage = .English
```

Comment:
The default language should be the `.BrowserDefault` I mentioned above.

Reply:
Updated